Library/Homebrew/utils/ruby.sh

```diff
@@ -36,12 +36,10 @@ setup-ruby-path() {
 
       if [[ -n "$HOMEBREW_RUBY_PATH" ]]
       then
-        ruby_version_major="$("$HOMEBREW_RUBY_PATH" --version)"
-        ruby_version_major="${ruby_version_major#ruby }"
-        ruby_version_major="${ruby_version_major%%.*}"
+        ruby_version="$("$HOMEBREW_RUBY_PATH" -e "puts RUBY_VERSION")"
       fi
 
-      if [[ "$ruby_version_major" != "2" || -n "$HOMEBREW_FORCE_VENDOR_RUBY" ]]
+      if [[ "$ruby_version" != "2.3.3" || -n "$HOMEBREW_FORCE_VENDOR_RUBY" ]]
```

Comment:
Does this mean that if the ruby at HOMEBREW_RUBY_PATH isn't precisely 2.3.3, that vendor is going to be forced?

Reply:
Yes.